meta-fobnail-distro/recipes-fobnail/fobnail-attester/fobnail-attester_git.bb

```diff
@@ -0,0 +1,28 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1ff626c483f6f10b69ee67274daf0775"
+
+SRC_URI = " \
+    git://github.com/fobnail/fobnail-attester;protocol=https;branch=main \
+    file://0001-Makefile-fix-to-build-in-Yocto.patch \
+    file://fobnail_usb.network \
+"
+
+PROVIDES:${PN} += "fobnail-attester"
+
+DEPENDS = "openssl tpm2-tss libcoap qcbor"
+
+PV = "1.0+git${SRCPV}"
+SRCREV = "0b7085ff80a3417d079888c88f10578e8120ce88"
+
+S = "${WORKDIR}/git"
+NETWORKCONFDIR = "/etc/systemd/network"
```

Comment:
@kotylamichal we should use bitbake.conf file from poky for path variables https://git.yoctoproject.org/poky/tree/meta/conf/bitbake.conf?h=honister so it should be `$sysconfdir/systemd/network`

Reply:
done, but it was moved to `systemd-conf_%.bbappend`